.github/workflows/pyinstaller-windows.yml

```
@@ -18,22 +18,27 @@ jobs:
     - uses: actions/checkout@v2
       with: { submodules: recursive }
     - uses: conda-incubator/setup-miniconda@v2
-      with: { mamba-version: "*", channels: "conda-forge", channel-priority: true, python-version: "3.10" }
-    - name: install dependencies
+      with:
+        python-version: "3.10"
+        miniforge-variant: Mambaforge
+        channels: conda-forge,defaults
+        channel-priority: true
+        activate-environment: autotag-metadata
+        environment-file: environment.yml
```

```suggestion
      with:
        python-version: "3.10"
        miniforge-variant: Mambaforge
        environment-file: environment.yml
```

I think this is good enough here.

---

You are right. This is indeed not necessary.